triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -86,6 +91,13 @@ trait AbstractTriggerServiceTestHelper
     DarReader.assertReadArchiveFromFile(darPath).map(p => p.pkgId -> p.proto)
   protected lazy val testPkgId: PackageId = dar.main._1
 
+  private[this] val triggerRunnerLogAppender = new ListAppender[ILoggingEvent]()
+  private[this] val triggerRunnerLogger: Logger =
+    LoggerFactory.getLogger(classOf[Runner]).asInstanceOf[Logger]
+
+  triggerRunnerLogAppender.start()
+  triggerRunnerLogger.addAppender(triggerRunnerLogAppender)
```

Comment:
Code to snag log events from the trigger runner (current test code only snags log events from the trigger service)